setup.cfg

```diff
@@ -28,6 +28,8 @@ install_requires =
     # Schema definition 
     pydantic<2
     # Pickling library
+    cloudpickle==2.2.1
```

Comment:
**question**: Is there a good reason to keep the requirement so strict?

Reply:
We ran into an issue with `dill` where the internal changed between `0.3.4` and `0.3.6` and some environments where installing the old version for some reason.

Because of the change, some pickles were failing to be unpickled at runtime.

This requirement pin is a preemptive move to prevent this happening with `cloudpickle`.

Reply:
Cloudpickle and dill are likely to be used by other packages our users might use alongside the SDK, eg scikit-learn. We need to be wary of dependency conflicts reported by users.